scripts/novaseq/demux-novaseq.bash

```diff
@@ -15,14 +15,15 @@ PROJECTLOG=${4?'projectlog needed'}
 
 RUN=$(basename ${IN_DIR})
 OUT_DIR=${DEMUXES_DIR}/${RUN}
-SCRIPT_DIR=/home/proj/${ENVIRONMENT}/bin/git/demultiplexing/scripts/novaseq/
 EMAIL=clinical-demux@scilifelab.se
 
 SLURM_ACCOUNT=development
 if [[ ${ENVIRONMENT} == 'production' ]]; then
     SLURM_ACCOUNT=production
 fi
 
+SCRIPT_DIR=/home/proj/${ENVIRONMENT}/bin/git/demultiplexing/scripts/novaseq/
+
```

Comment:
Dont think this was needed - we also don't actually use this script since we do the demultiplexing of novaseq through `cg`

Reply:
Removed this from the pull request.